src/core/utils/build.js

```diff
@@ -0,0 +1,19 @@
+/* @flow */
+
+type GetDeploymentVersionParams = {|
+  versionJson: {
+    commit: string,
+    version?: string,
+  },
+|};
+
+export const getDeploymentVersion = ({
+  versionJson,
+}: GetDeploymentVersionParams): string => {
+  const deploymentVersion =
```

Comment:
Just trying to understand this logic. We are preferring `version` to `commit`. Is that because we expect to need this mostly on prod, where `version` will tell us which pushed version we're on? I ask because it seems like `commit` is more granular and therefore I'm a bit confused about why we're not preferring `commit`.

Reply:
Yeah, exactly. I want to favor a tag over a commit in stage/prod because we'll be able to track regressions, hopefully. Our workflow (and everything else) is based on the notion of "milestone", which is what `version` stores.